library/CMakeLists.txt

```diff
@@ -124,7 +129,7 @@ set(CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "\${CPACK_PACKAGING_INSTALL_PR
 # Package name
 set(package_name rocsparse)
 
-set(ROCSPARSE_CONFIG_DIR "\${CPACK_PACKAGING_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}" CACHE PATH "Path placed into ldconfig file")
+set(ROCSPARSE_CONFIG_DIR "/opt/rocm/${CMAKE_INSTALL_LIBDIR}" CACHE PATH "Path placed into ldconfig file")
```

Comment:
I do not agree that /opt/rocm/ should be hard coded.

Reply:
This is kept /opt/rocm as we want the normal users to use the ldconfig path to find the dependent libs with a symlink(/opt/rocm -> /opt/rocm-ver) and advanced users will use the RUNPATH to find all the dependent libs